packages/@react-aria/datepicker/intl/sr-SP.json

```diff
@@ -1,17 +1,18 @@
 {
-  "calendar": "Kalendar",
-  "day": "дан",
-  "dayPeriod": "пре подне/по подне",
-  "endDate": "Datum završetka",
-  "era": "ера",
-  "hour": "сат",
-  "minute": "минут",
-  "month": "месец",
-  "second": "секунд",
-  "startDate": "Datum početka",
-  "year": "година",
-  "weekday": "дан у недељи",
-  "timeZoneName": "временска зона",
-  "selectedDateDescription": "Izabrani datum: {date}",
-  "selectedRangeDescription": "Izabrani opseg: od {startDate} do {endDate}"
+  "calendar": "Calendar",
+  "day": "day",
+  "dayPeriod": "AM/PM",
+  "endDate": "End Date",
+  "era": "era",
+  "hour": "hour",
+  "minute": "minute",
+  "month": "month",
+  "second": "second",
+  "selectedDateDescription": "Selected Date: {date}",
+  "selectedRangeDescription": "Selected Range: {startDate} to {endDate}",
+  "selectedTimeDescription": "Selected Time: {time}",
+  "startDate": "Start Date",
+  "timeZoneName": "time zone",
+  "weekday": "day of the week",
+  "year": "year"
```

Comment:
Looks like this one didn't get updated?

Reply:
Sorry, missed that one, fixed! I checked others and look good,